climada/entity/exposures/litpop/litpop.py

```diff
@@ -190,25 +193,26 @@ def set_countries(self, countries, res_arcsec=30, exponents=(1,1),
 
         try:
             rows, cols, ras_trans = u_coord.pts_to_raster_meta(
-                (self.gdf.longitude.min(), self.gdf.latitude.min(),
-                 self.gdf.longitude.max(), self.gdf.latitude.max()),
-                u_coord.get_resolution(self.gdf.longitude, self.gdf.latitude))
-            self.meta = {
+                (exp.gdf.longitude.min(), exp.gdf.latitude.min(),
+                 exp.gdf.longitude.max(), exp.gdf.latitude.max()),
+                u_coord.get_resolution(exp.gdf.longitude, exp.gdf.latitude))
+            exp.meta = {
                 'width': cols,
                 'height': rows,
-                'crs': self.crs,
+                'crs': exp.crs,
                 'transform': ras_trans,
             }
         except ValueError:
             LOGGER.warning('Could not write attribute meta, because exposure'
                            ' has only 1 data point')
-            self.meta = {'crs': self.crs}
-        self.check()
+            exp.meta = {'crs': exp.crs}
+        exp.check()
+        return exp
 
     def set_nightlight_intensity(self, countries=None, shape=None, res_arcsec=15,
                         reference_year=DEF_REF_YEAR, data_dir=SYSTEM_DIR):
         """
-        Wrapper around `set_countries` / `set_custom_shape`.
+        Wrapper around `from_countries` / `set_custom_shape`.
```

Comment:
Replace all occurrences of the deprecated `set_`-method by the new constructor, including comments, docstrings, examples, tests and tutorials.